.github/workflows/ci.yml

```diff
@@ -106,6 +140,16 @@ jobs:
           distro \
           pytest
 
+    - name: Install deps on MacOS
+      if: runner.os == 'macOS'
+      run: |
+        # Install deps
+        ##sudo apt update
+        brew install git gcc gnupg python@3 
+        conda install -y -q -c conda-forge \
```

Comment:
Can we get away from conda, and use only brew?

Reply:
Since its python libraries, so used pip command rather than brew.